### DISCUSSION TITLE
Launching remote app creates a duplicate user log-in

### DISCUSSION THREAD

**Discussion by j-omega** [kimmknight/remoteapptool#108] [Q&A]
I am hosting my remote apps on Windows 10 Professional 22H2.

When I connect to an app from a client PC, the app launches as expected. However, if I am already logged into the host PC, the remote app is started in a new (duplicate) user session. See the attached image below.

This is not the desired behavior. Ideally, if I am already logged into the PC, the RemoteApp connection would be made to the existing user session.

Is the behavior that I am observing the expected behavior, or is it due to a misconfiguration on my end?

![duplicate-user](https://github.com/kimmknight/remoteapptool/assets/1540706/1a301916-99c5-4dca-bdf7-71bb651d6f26)

**Comment by kimmknight** [comment]
Try adding the following line to your RDP file:

```
administrative session:i:1
```

This will force console mode, which should take over the locally logged-on user session. It is the same as running:

```
mstsc.exe /console
```

I have added issue #100 (feature request) to have this be an option in Create Client Connection.

**Comment by kimmknight** [comment]
And to answer your question @j-omega, yes, what you seem to be experiencing is normal/expected behaviour for an RDP session.

**Comment by j-omega** [comment]
I added the line, but the RemoteApp connection still creates a duplicate session. I should also note that a regular remote desktop session connects me to the console session. I wonder if this is related to the fact that Windows 10 (not Window Server) is the RemoteApp host.

**Comment by kimmknight** [comment]
Ok, thanks for the update @j-omega 

There are other RDP file options and local/group policy options that might affect this behaviour.

For a list of RDP file options, check out my RDP File Editor:

https://kimmknight.github.io/rdpfileeditor/

Let us know if you find a solution.

**Comment by sashaqwert** [comment]
> I added the line, but the RemoteApp connection still creates a duplicate session.

Windows starts a separate session for RemoteApp. Otherwise, Explorer.exe from the remote PC would overlap the local explorer.exe.
If you want to avoid session duplication, there are 2 solutions:
1. Disable automatic last user logon (this method will mainly help if the computer is not always used locally): https://admx.help/?Category=Windows_11_2022&Policy=Microsoft.Policies.WindowsLogon2::AutomaticRestartSignOn

2. Try use program launch in a session instead of RemoteApp:
This can be done by adding the line to the RDP file
```
alternate shell:s:1.exe
```
(But it's better not to do that...)
